.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
```

Comment:
Aren't you using prettier? That should actually cover this already.

Reply:
yes but IDEs just get it wrong without this config

Reply:
(before you manually run prettier)

Reply:
If you'd like i move that into an extra pull